BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//brave/build/cargo.gni")
+
+challenge_bypass_build_path = "$root_gen_dir/challenge_bypass_ristretto/out"
+
+challenge_bypass_lib_path = challenge_bypass_build_path
+if (is_debug) {
+  challenge_bypass_lib_path += "/debug/incremental"
```

Comment:
is this the right path? I would think it should only be `/debug`

Reply:
Path was wrong :-)